Installation/Mapping/vrijbrp.dossierToZaakDetail.mapping.json

```diff
@@ -0,0 +1,35 @@
+    "status": "{% if dossier.status|default %}{% if dossier.dossierId|default %}{% set status = dossier.status|merge({'dossierId': dossier.dossierId}) %}{% else %}{% set status = dossier.status %}{% endif %}{{ map('https:\/\/commongateway.nl\/mapping\/vrijbrp.statusToZGWStatus.mapping.json', status)|json_encode }}{% endif %}",
+    "eigenschappen": "[{\"naam\": \"VERHUISDATUM\", \"waarde\": \"{{ dossier.startDate }}\", \"eigenschap\": {\"_sourceId\": \"VERHUISDATUM\", \"naam\": \"VERHUISDATUM\"}},{\"naam\": \"STRAATNAAM_NIEUW\", \"waarde\": \"{{ newAddress.street }}\", \"eigenschap\": {\"_sourceId\": \"STRAATNAAM_NIEUW\", \"naam\": \"STRAATNAAM_NIEUW\"}},{\"naam\": \"HUISNUMMER_NIEUW\", \"waarde\": \"{{ newAddress.houseNumber }}\", \"eigenschap\": {\"_sourceId\": \"HUISNUMMER_NIEUW\", \"naam\": \"HUISNUMMER_NIEUW\"}},{\"naam\": \"HUISLETTER_NIEUW\", \"waarde\": \"{{ newAddress.houseLetter }}\", \"eigenschap\": {\"_sourceId\": \"HUISLETTER_NIEUW\", \"naam\": \"HUISLETTER_NIEUW\"}},{\"naam\": \"TOEVOEGINGHUISNUMMER_NIEUW\", \"waarde\": \"{{ newAddress.houseNumberAddition }}\", \"eigenschap\": {\"_sourceId\": \"TOEVOEGINGHUISNUMMER_NIEUW\", \"naam\": \"TOEVOEGINGHUISNUMMER_NIEUW\"}},{\"naam\": \"POSTCODE_NIEUW\", \"waarde\": \"{{ newAddress.postalCode }}\", \"eigenschap\": {\"_sourceId\": \"POSTCODE_NIEUW\", \"naam\": \"POSTCODE_NIEUW\"}},{\"naam\": \"WOONPLAATS_NIEUW\", \"waarde\": \"{{ newAddress.residence }}\", \"eigenschap\": {\"_sourceId\": \"WOONPLAATS_NIEUW\", \"naam\": \"WOONPLAATS_NIEUW\"}},{% if newAddress.mainOccupant|default and newAddress.mainOccupant.bsn|default %}{\"naam\": \"BSN_HOOFDBEWONER\", \"waarde\": \"{{ newAddress.mainOccupant.bsn }}\", \"eigenschap\": {\"_sourceId\": \"BSN_HOOFDBEWONER\", \"naam\": \"BSN_HOOFDBEWONER\"}},{% endif %}{% if previousMunicipality.code|default %}{\"naam\": \"GEMEENTECODE\", \"waarde\": \"{{ previousMunicipality.code }}\", \"eigenschap\": {\"_sourceId\": \"GEMEENTECODE\", \"naam\": \"GEMEENTECODE\"}},{% endif %}{% set index = 0 %}{% for relocator in relocators %}{\"naam\": \"MEEVERHUIZENDE_GEZINSLEDEN.MEEVERHUIZEND_GEZINSLID.{{ index }}.BSN\", \"waarde\": \"{{ relocator.bsn }}\", \"eigenschap\": {\"_sourceId\": \"MEEVERHUIZENDE_GEZINSLEDEN.MEEVERHUIZEND_GEZINSLID.{{ index }}.BSN\", \"naam\": \"MEEVERHUIZENDE_GEZINSLEDEN.MEEVERHUIZEND_GEZINSLID.{{ index }}.BSN\"}},{\"naam\": \"MEEVERHUIZENDE_GEZINSLEDEN.MEEVERHUIZEND_GEZINSLID.{{ index }}.ROL\", \"waarde\": \"{{ relocator.declarationType }}\", \"eigenschap\": {\"_sourceId\": \"MEEVERHUIZENDE_GEZINSLEDEN.MEEVERHUIZEND_GEZINSLID.{{ index }}.ROL\", \"naam\": \"MEEVERHUIZENDE_GEZINSLEDEN.MEEVERHUIZEND_GEZINSLID.{{ index }}.ROL\"}},{% endfor %}{\"naam\": \"AANTAL_PERS_NIEUW_ADRES\", \"waarde\": \"{{ newAddress.numberOfResidents }}\", \"eigenschap\": {\"_sourceId\": \"AANTAL_PERS_NIEUW_ADRES\", \"naam\": \"AANTAL_PERS_NIEUW_ADRES\"}}]"
```

Comment:
This looks weird / not right?

Reply:
It is not an optimal mapping, that I grant you, but this is tested against the eigenschappen we have in the current SimXML to ZGW bundle